guacamole/src/main/webapp/app/navigation/templates/guacMenu.html

```diff
@@ -0,0 +1,7 @@
+<div class="menu-dropdown" ng-class="{open: menuShown}" ng-click="toggleMenu()">
+    <div class="menu-title">{{menuTitle}}</div>
+    <div class="menu-indicator"></div>
+
+    <!-- Menu options -->
+    <div class="options" ng-transclude></div>
```

Comment:
The naming itself comes from its use within the `guacUserMenu` directive, which is where the meat of the `guacMenu` directive comes from.

Now that the menu is all abstract, it might make more semantic sense for this to be `menu-contents`.